migration-guide.md

```diff
@@ -86,13 +89,77 @@ Boolean props changes from `is<X>` to `<x>`:
 
 - `Divider` component was renamed to `Separator`, the props stay the same.
 
+### Removed components
+- `Show` and `Hide` components were removed. Use `hideFrom` and `hideBelow` props available on all components to control visibility instead.
+- `UnorderedList` and `OrderedList` components were removed. Use `List` component with `as="ul"` and `as="ol"` properties instead.
+
 ### Namespace style components
 
-The following components now use the `namespace` style imports:
+The following components now use the `namespace` style imports[^1]:
```

Comment:
Not for this PR more to check can we use the `@deprecated` directive so they show up in the projects as deprecated